PartnerMeeting.Rmd

```diff
@@ -0,0 +1,26 @@
+---
+title: "Partner meeting"
+author: "Vizlab, Data Science Branch, IIDD"
+output: html_document
+---
+
+### Interactive map
```

Comment:
Let's skip this extra header since the file is pretty short. I'd also suggest adding to the text to guide the reviewers a little bit, for example adding "The purpose of this document is to establish agreed upon spatial extent for each of the saline lakes, including:
1. Lake polygons
2. Watershed boundaries
3. Flowlines within each watershed

The map below can be panned and zoomed to verify the delineations are accurate to each lake. Clicking on polygons will provides identifying information about each watershed. Please provide feedback in this [linked] spreadsheet if the mapped data require changes. 

Use the toggles on the righthand corner to turn on or off different data layers. These include preliminary gage sites that are associated with each lake.